static/openapi/prerendering-spec.json

```diff
@@ -3060,1025 +3384,1091 @@
           },
           "400": {
-            "description": "Bad request",
+            "description": "Bad request.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Bad request",
+                  "description": "Bad request.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Error status"
+                      "description": "Error status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Error message"
+                      "description": "Error message."
                     }
                   },
-                  "required": ["status", "message"]
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Header x-photoniq-customerid is required"
+                  }
                 }
               }
             }
           },
           "500": {
-            "description": "Internal server error",
+            "description": "Internal server error.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Internal server error",
+                  "description": "Internal server error.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Error status"
+                      "description": "Error status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Error message"
+                      "description": "Error message."
                     }
                   },
-                  "required": ["status", "message"]
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Internal server error"
+                  }
                 }
               }
             }
           }
         }
       },
-      "delete": {
-        "summary": "Delete origin",
-        "tags": ["Origins"],
-        "description": "Deletes an origin.",
+      "patch": {
+        "summary": "Update prefetch configuration",
+        "tags": ["Prefetch"],
+        "description": "Updates the prefetch configuration associated with the origin.",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "cronSchedule": {
+                    "type": "string",
+                    "description": "Cron expression for scheduling."
```

Comment:
Can you provide "examples:" for all the _requestBody_ entries.  Would be good to do it for all requestBody.
I don't know if people know how to do "Cron expressions".

Reply:
We started to add response examples. Soon, we're also going to add request examples.

Reply:
I just added request body examples.